java/src/hdf/hdf5lib/HDF5Constants.java

```diff
@@ -158,15 +158,15 @@
     public static final long H5E_ALREADYINIT = H5E_ALREADYINIT();
     /** Major error codes - Invalid arguments to routine */
     public static final long H5E_ARGS = H5E_ARGS();
-    /** Major error codes - Object atom */
+    /** Major error codes - Object ID */
```

Comment:
I changed the text to say "ID" even though there's nothing we can do about the actual code.